### PR TITLE
MaMuJoCo Doc Pages updates

### PR DESCRIPTION
**Pull request by Kallinteris-Andreas**
# Description

Mainly expanded `#version-history`, but also includes some typo fixes

### CODE REVIEW

File: docs/envs/MaMuJoCo/ma_ant.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ If partitioning, is None then the environment contains a single agent with the s
 | 5   | Torque applied on the rotor between the front right two links     | -1          | 1           | angle_2 (front_right_leg)        | hinge | torque (N m) |
 | 6   | Torque applied on the rotor between the torso and back left hip   | -1          | 1           | hip_3 (back_leg)                 | hinge | torque (N m) |
 | 7   | Torque applied on the rotor between the back left two links       | -1          | 1           | angle_3 (back_leg)               | hinge | torque (N m) |
-### if partitioning == "2x4":  # neighbouring legs together
+### if partitioning == "2x4":  # neighboring legs together (front and back)
 ```{figure} Ant_2x4.png
```
Comment (rodrigodelazcano) [inline docs/envs/MaMuJoCo/ma_ant.md:44]: I currently can't visualize this image. Is it required?
https://robotics.farama.org/envs/MaMuJoCo/ma_ant/#if-partitioning-2x4-neighbouring-legs-together

**Review by rodrigodelazcano — COMMENTED** [review]
Looks good. I left a review and please remove these [lines](https://github.com/Farama-Foundation/Gymnasium-Robotics/blob/5b3d624a88013cb13ac0a958d27846517de587eb/docs/envs/MaMuJoCo/ma_ant.md?plain=1#L172) from the `.md` files of each individual environment. These will remove the tab option in the webpage.

Also, can you make another PR removing the `ma_mujoco` dependencies install from `setuptools` and the respective documentation?

File: docs/envs/MaMuJoCo/ma_ant.md (hunk 1 above)
Comment (Kallinteris-Andreas) [reply docs/envs/MaMuJoCo/ma_ant.md:44]: I added it, do no know why it was missing from repo.
I will be adding more images soonish ^ TM

**Comment by Kallinteris-Andreas** [comment]
"""Also, can you make another PR removing the ma_mujoco dependencies install from setuptools and the respective documentation?"""
you mean disabling `pip install gymnasium_robotics[mamujoco]`?

**Comment by rodrigodelazcano** [comment]
> """Also, can you make another PR removing the ma_mujoco dependencies install from setuptools and the respective documentation?""" you mean disabling `pip install gymnasium_robotics[mamujoco]`?

Correct. It doesn't add any functionality since all the dependencies are being installed with the normal installation process, including pettingzoo and jinja.

**Pull request merged**